util/syn_yosys.sh

```diff
@@ -35,41 +35,27 @@ cd syn_out
 grep -Ev 'incdir|pins_if' ../build/*/*/*.scr | sed 's!.*/!!' > flist_gold
 
 # generate revised flist by replacing '.sv' by '.v' and removing packages
-sed 's/.sv/.v/g' flist_gold | grep -v "_pkg.v" > flist_rev
+sed -e 's/.sv/.v/g' flist_gold | grep -v "_pkg.v" > flist_rev
 
 #-------------------------------------------------------------------------
 # convert all RTL files to Verilog
 #-------------------------------------------------------------------------
 
-# hack SystemVerilog files to avoid SV2V and/or LEC issues
-# TODO: eventually remove these hacks
-
-# 1) Setting ByteAccess = 0 for tlul_adapter_sram.sv doesn't work. This
-# hack changes the functionality so needs fixing
-sed -i 's/ByteAccess *(0)/ByteAccess(1)/g' *.sv
-
-# 2) Replace "inside" for dm_csrs.sv and dm_mem.sv. This hack changes
-# the functionality so needs fixing
-sed -i 's/) inside/)/g' dm_csrs.sv
-sed -i 's/\[(dm::Data0):DataEnd\]:/dm::Data0:/g' dm_csrs.sv
-sed -i 's/\[(dm::ProgBuf0):ProgBufEnd\]:/dm::ProgBuf0:/g' dm_csrs.sv
-sed -i 's/inside//g' dm_mem.sv
-sed -i 's/\[(dm::DataAddr):DataEndAddr\]:/dm::DataAddr:/' dm_mem.sv
-sed -i 's/\[DataBaseAddr:DataEndAddr\]:/DataBaseAddr:/' dm_mem.sv
-sed -i 's/\[ProgBufBaseAddr:ProgBufEndAddr\]:/ProgBufBaseAddr:/' dm_mem.sv
-sed -i 's/\[AbstractCmdBaseAddr:AbstractCmdEndAddr\]:/AbstractCmdBaseAddr:/' dm_mem.sv
-sed -i 's/\[FlagsBaseAddr:FlagsEndAddr\]:/FlagsBaseAddr:/' dm_mem.sv
-
 printf "\nSV2V VERSION:\n"
 sv2v --version
 
 printf "\nSV2V ERRORS:\n"
 
+# drive strengths are not supported by Yosys
+sed -i.bak -e "s/VERILATOR/SYNTHESIS/" prim_generic_pad_wrapper.sv
+
 # prim_util_memload.sv is only meant to be included within a module
 mv prim_util_memload.sv{,h}
-sed -i "s/prim_util_memload\.sv/prim_util_memload.svh/" *.sv
+sed -i.bak -e "s/prim_util_memload\.sv/prim_util_memload.svh/" *.sv
```

Comment:
I've opened https://github.com/lowRISC/opentitan/pull/2977 to get this file renamed. If you come across such problems, please file an issue to get them sorted out.